charters/wot-wg-2023-draft.html

```diff
@@ -491,6 +491,81 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 
         <section>
           <h3 id="external-coordination">External Organizations</h3>
+          <p>
+            To succeed in establishing inter-platform standards,
+            W3C needs to coordinate with IoT alliances and standards development organizations.
+          </p>
+          <p>
+            List of new liaisons we want to form
+          </p>
+          <dl>
+            <dt> <a href="https://opcfoundation.org/">OPC Foundation</a>
+              <dd>
+                For coordination on the development of the OPC-UA Binding for the W3C Web of Things via the joint working group.
+                A Memorandum of Understanding (MoU) / Multi-Organization Cooperation Agreement (MOCA) will be signed.
```

Comment:
unfortunately, I don't think we can say "will" here; this would commit the W3C.  Perhaps we can say "seek a formal relationship for joint work"

Reply:
kaz mentioned in the meeting that we should also look at SDW/OGC collaboration for an example for how to organize joint work and liaisons

Reply:
> unfortunately, I don't think we can say "will" here; this would commit the W3C. Perhaps we can say "seek a formal relationship for joint work"

How about:
A Memorandum of Understanding (MoU) / Multi-Organization Cooperation Agreement (MOCA) will be considered in order to establish a formal relationship.

I wanted to keep the MoU and MOCA wording in there but if it is not needed, I can remove entirely.

Reply:
> kaz mentioned in the meeting that we should also look at SDW/OGC collaboration for an example for how to organize joint work and liaisons

I had a look at this and realized that there is a whole chapter about OGC collaboration with its different groups. (see https://www.w3.org/2021/10/sdw-charter.html#coordination). Also, they are pretty specific about the fact that TAG equivalent of OGC will provide guidance. If we need to go into detail at this level, we need more discussion. My personal opinion is that we do not need this level of detail unless we really need different type of collaboration with different groups of the related SDO.

Reply:
In my view this is not needed since there will be no several sub-groups involved.

Reply:
```suggestion
                A Memorandum of Understanding (MoU) / Multi-Organization Cooperation Agreement (MOCA) will be considered in order to establish a formal relationship.
```